apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -1753,7 +1753,7 @@
   "DeviceAction": {
     "allowAppPermission": "Open the {{wording}} app on your device",
     "allowAppPermissionSubtitleToken": "to manage your {{token}} tokens",
-    "allowManagerPermission": "Allow Ledger Manager on your device",
+    "allowManagerPermission": "Allow Ledger Manager on your {{productName}}",
```

Comment:
[ASK] Will it change after next Smartling pull ? Moreover we need to know if the other translation contains the placeholders with the same attribute name ?